src/views/ContactView.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { Helmet } from "react-helmet-async";
+import { useSelector } from "react-redux";
+
+import ScreenSplit from "../components/ScreenSplit";
+import ContactUs from "../components/ContactUs";
+
+const ContactView = () => {
+  const { contact } = useSelector((state) => state.appData);
```

Comment:
small one, but lets rename state to contacts

Reply:
i change the state in testimonialsSlider to testimonials aswell then 👍

Reply:
cool))